### PR TITLE
Add integration tests for exporter and collector

### PR DESCRIPTION
**Pull request by zyiou**

### CODE REVIEW

File: pkg/test/integration_test.go [hunk 1]
```diff
@@ -0,0 +1,153 @@
+// Copyright 2020 VMware, Inc.
```
Comment (srikartati) [inline pkg/test/integration_test.go:1]: Filename could be different. integration_test seems very generic. exporter_collector_test?

Comment (zyiou) [reply pkg/test/integration_test.go:1]: Makes sense. Thanks

**Review by srikartati — COMMENTED** [review]
LGTM. As this is dependent on other collector PR. Lets merge that one first and this can be merged later.

File: pkg/test/exporter_collector_test.go [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+	assert.Equal(t, []uint16{101}, templateSet[registry.AntreaEnterpriseID], "TemplateSet does not store template elements (Antrea) correctly.")
+
+	dataSet, ok := dataMsg.Set.(entities.DataSet)
+	if !ok {
+		t.Error("Data packet is not decoded correctly.")
+	}
+	assert.Equal(t, []byte{1,2,3,4}, dataSet[registry.IANAEnterpriseID][8], "DataSet does not store elements (IANA) correctly.")
+	assert.Equal(t, uint64(12345678), dataSet[registry.ReverseEnterpriseID][1], "DataSet does not store reverse information elements (IANA) correctly.")
+	assert.Equal(t, "pod1", dataSet[registry.AntreaEnterpriseID][101], "DataSet does not store elements (Antrea) correctly.")
+}
```
Comment (srikartati) [inline pkg/test/exporter_collector_test.go:166]: can you add a new line?

File: pkg/test/exporter_collector_test.go [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+	"github.com/stretchr/testify/assert"
+	"k8s.io/klog/v2"
+
+	"github.com/vmware/go-ipfix/pkg/collector"
+	"github.com/vmware/go-ipfix/pkg/entities"
+	"github.com/vmware/go-ipfix/pkg/exporter"
+	"github.com/vmware/go-ipfix/pkg/registry"
+)
+
+type Addr struct {
```
Comment (srikartati) [inline pkg/test/exporter_collector_test.go:31]: why not directly use [net.Addr](https://golang.org/pkg/net/#Addr)?

Comment (zyiou) [reply pkg/test/exporter_collector_test.go:31]: Because net.Addr is an interface, I just implement that here.

Comment (srikartati) [reply pkg/test/exporter_collector_test.go:31]: ResolveUDPAddr and ResolveTCPAddr can help here. Those functions take a string as input and returns net.Addr interface.

Comment (zyiou) [reply pkg/test/exporter_collector_test.go]: Got it. Updated. Thanks!

**Review by srikartati — APPROVED** [review]

**Pull request merged**